tests/integration/sdk_api/alerts/test_alerts.py

```diff
@@ -176,6 +176,82 @@ def test_alert_after_project_deletion(self):
         )
         self._post_event(alert_entity_project, event_name, alert_entity_kind)
 
+    def test_alert_templates(self):
+        project_name = "my-project"
+        project = mlrun.new_project(project_name)
+
+        # one of the pre-defined system templates
+        drift_system_template = mlrun.common.schemas.AlertTemplate(
+            template_id=2,
+            template_name="DriftDetected",
+            template_description="Generic template for drift detected alerts",
+            system_generated=True,
+            description="Model drift has been detected",
+            severity=alert_constants.AlertSeverity.HIGH,
+            trigger={"events": [mlrun.common.schemas.alert.EventKind.DRIFT_DETECTED]},
+            reset_policy=mlrun.common.schemas.alert.ResetPolicy.MANUAL,
+        )
```

Comment:
Why not import the `pre_defined_templates` directly, and take the definitions from the source instead of re-defining it here? This way you can verify that you get what is defined, even if we modify the pre-defined templates at some point.